dcos/subcommand.py

```diff
@@ -121,6 +145,10 @@ def distributions():
             if os.path.isdir(
                 os.path.join(
                     subcommand_dir,
+                    subdir)) or
+            os.path.isdir(
+                os.path.join(
+                    subcommand_dir,
                     subdir,
                     constants.DCOS_SUBCOMMAND_VIRTUALENV_SUBDIR))
```

Comment:
Should we do the same thing for binary CLIs? I think we added the `env` folder so that we can store `package.json` without overriding anything that `virtualenv` does. I think we need to do the same for the `zip` case. That would simplify this code and the code that finds the executable.